runner/client/src/mill/runner/client/MillProcessLauncher.java

```diff
@@ -114,12 +114,17 @@ static boolean isWin() {
   }
 
   static String javaHome() throws IOException {
-    String jvmId;
+    String jvmId = null;
     Path millJvmVersionFile = millJvmVersionFile();
 
     String javaHome = null;
     if (Files.exists(millJvmVersionFile)) {
       jvmId = Files.readString(millJvmVersionFile).trim();
+      if (jvmId.equals("global")) jvmId = null;
```

Comment:
FYI in coursier, `system` stands for "use the already installed / available JVM". Don't know which is better, if `global` feels natural to enough people, I could accept it in coursier too.

Reply:
`system` works too, I was unaware of Coursier's default. I can change this over